spec/features/phone/default_phone_selection_spec.rb

```diff
@@ -74,11 +74,13 @@
         user.reload
 
         expect(page).to have_current_path(account_path)
-
-        node = page.first('.grid-row > .grid-col-5', text: new_phone)
-        expect(node).to have_content '202-555-3111'
-        parent = node.first(:xpath, './/..')
-        expect(parent).to have_content t('account.index.default')
+        expect(page).to have_xpath(
+          "//h2[contains(text(),'#{t('account.index.phone')}')]" \
+            "/.." \
+            "//*[contains(@class,'grid-row')]" \
+            "//*[contains(text(),'#{new_phone}')]" \
+            "[contains(text(),'(#{t('account.index.default')})')]",
+        )
```

Comment:
Not proud of this 😆  Will try to give it fresh eyes on Monday, but open to suggestions.